config/routes.rb

```diff
@@ -47,6 +47,9 @@
     to: "services_and_information#index",
     as: :services_and_information
 
+  get "/prepare-eu-exit-live-UK",
```

Comment:
Should this be `/prepare-eu-exit-live-uk`? It doesn't make sense to capitalise UK but not EU.